### ISSUE TITLE
Please suggest the Editor to use for the C# code elements. Please consider producing a version of this tutorial to follow inside of VS

### ISSUE THREAD

**Issue by rstannard** [dotnet/docs#6218]
I've just completed following this tutorial and found it very helpful. I would just make the following comments how it could be slightly improved.
1/ please advise the reader that this tutorial is aimed to be followed outside of the Visual Studio 2017 environment. 
2/ if (1) is the case then please suggest an editor to use for completing the code in.
I followed the tutorial using a mix of following the command line steps to create the project structure as well as VS2017 for the C# code pieces.
3/ it would be very useful to see a version of this tutorial which shows the steps to follow inside of VS.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: e0063ceb-57ea-f30c-efed-b2a36314ccaa
* Version Independent ID: bd9a321e-2ef2-491d-cd37-2d9c66f90cc9
* Content: [Unit testing C# with NUnit and .NET Core](https://docs.microsoft.com/en-us/dotnet/core/testing/unit-testing-with-nunit)
* Content Source: [docs/core/testing/unit-testing-with-nunit.md](https://github.com/dotnet/docs/blob/master/docs/core/testing/unit-testing-with-nunit.md)
* Product: **dotnet-core**
* GitHub Login: @rprouse
* Microsoft Alias: **mairaw**

**Comment by rprouse** [comment]
Good suggestions, thanks. You might want to change the title of this issue to something more meaningful since this repo is for all of the .NET Core documentation. I would do it, but I don't have edit rights in this repository.

**Comment by rstannard** [comment]
Hi Rob, I've just done that now - I hope its not too wordy ?

**Comment by skylord-a52** [comment]
Rather/as well as than using shell commands, which may be confusing to those new to Visual Studio (like myself), I'd recommend including in-GUI ways to do the same thing. 

`dotnet new -i NUnit3.DotNetNew.Template` can be avoided by installing the NUnit VS Packages from the Marketplace within Visual Studio, for example, and adding new solutions and linking references are fairly simple to do from within the Visual Studio UI.

**Comment by mairaw** [comment]
Thanks for your feedback @rstannard. I'm adding this to our backlog. We usually start the topics with the CLI version since it's cross-platform, but indeed topics using VS would also be useful.

/cc @BillWagner

**Comment by leder11011** [comment]
I strongly discourage using VS or Code as long as they are not helpful. I suggest .NET core for Linux and the bash CLI in conjunction with gedit, atom, vim or some other Linux editor available.

**Comment by Rick-Anderson** [comment]
@Thraka recommend closing this issue. VS is fine as the editor. This tutorial is consistent with all the rest of them.

Unit testing is generally not the onboarding/most beginning tutorial.

**Comment by Thraka** [comment]
@mairaw  owns this article so I would let her make the call, but I agree. It will all be rewritten anyhow when these get remade.

**Comment by StingyJack** [comment]
@Rick-Anderson  yeah but someone who is skilled with VS and familiar with unit testing cant actually follow these instructions and get a working result. at least not without monkeying and web searching a bit. For example I would never think to choose a console application as the project type, but Nunit seems to need that.  The doc page even says 

> You've built a small library and a set of unit tests for that library

Then again I wouldnt bother typing most of the stuff in these "tutorials" when I can hit 3 or 4 keys in a VS shortcut.  It may be a GUI but its still only 3 or 4 keys compared to typing "dotnet <anything>".

**Comment by adegeo** [comment]
@BillWagner @IEvangelist I think this one should be reopened. We really should have parity on these unit test articles for cli, visual studio, visual studio for mac.

Thoughts?

**Comment by IEvangelist** [comment]
> @BillWagner Bill Wagner FTE @IEvangelist David Pine FTE I think this one should be reopened. We really should have parity on these unit test articles for cli, visual studio, visual studio for mac.
> 
> Thoughts?

I think that we have consistent and uniform coverage so far. The articles require the SDK and your IDE of choice. I think that is sufficient, otherwise it becomes too repetitive. I believe that is the advantage to using the SDK, it is agnostic to IDE preference. If our instructions did not work, however; on a specific OS or IDE choice then we should fix it. I'm good with this one being closed. @BillWagner thoughts?

**Comment by sharwell** [comment]
Since a bot closed this and a user requested it be reopened, I'll do so now. Note that it may be closed again by a human per https://github.com/dotnet/docs/issues/6218#issuecomment-767516423.

**Comment by StingyJack** [comment]
@IEvangelist  - try following them with VS 2019 and assume you have only used MSFT's flagship unit testing platform before (since it has always been able to meet the need) and that you are not familiar with nunit or xunit.

**Comment by BillWagner** [comment]
Closing this as stale. The unit testing section has been substantially updated and reorganized since this was originally written.
